app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/conversationList.tsx

```diff
@@ -232,7 +232,7 @@ export const List = () => {
                   </Tooltip>
                 </TooltipProvider>
                 <div className="flex items-center gap-2">
-                  {searchParams.status === "closed" ? (
+                  {(searchParams.status === "closed" || searchParams.status === "spam") && (
```

Comment:
```suggestion
                  {searchParams.status !== "open" && (
```

maybe for consistency?